jest.config.ts

```diff
@@ -67,7 +67,9 @@ const config: Config = {
   // globalTeardown: undefined,
 
   // A set of global variables that need to be available in all test environments
-  // globals: {},
+  globals: {
+    CONFIG: {},
+  },
```

Comment:
**note:** Mocks config vars in the tests. We can always add useful mocks here later as needed.